rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -1161,6 +1165,37 @@ public String getJobHtml(@HeaderParam("sessionid") String sessionId, @PathParam(
         }
     }
 
+    @Override
+    @GET
+    @Path("jobs/{jobid}/xml")
+    @Produces("application/xml")
+    public InputStream getJobContent(@HeaderParam("sessionid") String sessionId, @PathParam("jobid") String jobId) {
+        try {
+            Scheduler s = checkAccess(sessionId, "/scheduler/jobs/" + jobId + "/livelog");
+
+            //            final Scheduler s = checkAccess(sessionId, "GEt jobs?jobid=" + jobId);
+            final Job jobContent = s.getJobContent(JobIdImpl.makeJobId(jobId));
+            return new Job2XMLTransformer().jobToxml((TaskFlowJob) jobContent);
+        } catch (NotConnectedRestException e) {
+            e.printStackTrace();
+        } catch (PermissionException e) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Combine this catch with the one at line 1179, which has the same body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2147)